inginious/frontend/templates/course_admin/submissions.html

```diff
@@ -426,7 +426,10 @@ <h4 class="modal-title">$:_("Download format")</h4>
 <!-- Submission table END -->
 
 </form>
-
+<script>
+    //Since we use post, we remove GET parameters in the URL to avoid inconsistency between POST and GET
+    window.history.replaceState({}, document.title, window.location.pathname);
```

Comment:
If you really want to avoid this, it's better not to parse the web.input in ``submissions.py`` in both GET and POST cases...Actually I don't think it's a real issue...